DoSomething/rogue
# Command to update signups and posts based on arguments passed in

#### What's this PR do?
- Updates `MakeSourceSms.php` command to be more general: 
  - Takes in three arguments to update signups AND posts by: 
    - `target`: The name of the field to update
    - `targetOldValue`: The value to search the target with.
    - `targetNewValue`: The value to update the target with.
    - (e.g. find all signups with the source (`target`) that equals `NULL` (`targetOldValue`) and update it to `phoenix-ashes` (`targetNewValue`).
- This script will allow us to update all signups and posts with the source values of `NULL` to `phoenix-ashes` since this is where they came from. This will make data more clean and accurate!

#### How should this be reviewed?
- Run the script (e.g. `php artisan rogue:updatefield --target=source --targetOldValue=NULL --targetNewValue=phoenix-ashes`)
- Make sure all previously `NULL` values for `source` are now `phoenix-ashes`

#### Relevant tickets
Fixes [https://www.pivotaltracker.com/n/projects/2019429/stories/155778318](this) Pivotal Card.

#### How to deploy
https://github.com/DoSomething/rogue/blob/master/docs/development/deployments.md
